### PR TITLE
MAINT-1711: apply Select2 lib

### PR DESCRIPTION
**Pull request by SashaMcr**
Change to select2 selector

### CODE REVIEW

**Comment by biplobice** [comment]
@SashaMcr 
I'm sorry, but could you please use Tom Select instead? This library should already be included in the core. It would also be preferable if you could use the Vue component. 

For reference, please check the following: https://github.com/concretecms/concretecms/pull/11210

Thank you.

File: controller.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace Concrete\Package\MdVisibilityAttribute;
 
+use Concrete\Core\Asset\AssetList;
```
Comment (biplobice) [inline controller.php]: @SashaMcr 
I think you no longer need the select2 CSS & JS. Please remove those files and this line.

Comment (SashaMcr) [reply controller.php]: @biplobice 

cleared

**Comment by biplobice** [comment]
Awesome! Thanks.

**Pull request merged**